src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1132,6 +1132,17 @@ def get_state(self) -> Any:
 
     getState = get_state
 
+    def set_state(self, state) -> Any:
+        """Set state of the command argument."""
+        parent_state = self.parent.get_state()
+        try:
+            parent_state[self.name] = state
+            self.parent.set_state(parent_state)
+        except KeyError:
```

Comment:
In which scenario will we get a KeyError?

If the KeyError happens due to some user error, we should throw the appropriate exception. If it never happens (guaranteed by the implementation), we should remove the try-except.